regress/unittests/win32compat/signal_tests.c

```diff
@@ -160,7 +160,7 @@ signal_test_wait_for_multiple_objects()
 			hObjects[i] = original_event;
 		}
 
-		TEST_RESOURCES(FALSE);
+		//TEST_RESOURCES(FALSE);
```

Comment:
@bingbing8 These were my invention to detect resource leaks.  Were these tests failing for you?  If so, I have some investigation to do.

Reply:
@NoMoreFood, yes, test failed on win12r2 especially on CI system. please see previous commit in this PR. I disable it for troubleshooting purpose. If you have idea why it fails, that would be helpful.

Reply:
@bingbing8 Thanks, I'll investigate.

Reply:
@bingbing8 This looks like a consistency issue with the value that GetProcessHandleCount() returns, specifically on Windows 8.1 / 2012 R2.  So for now it seems to be fine to keep it commented out.   Do you happen to know anyone in the kernel development area?  If so, could you see if the attached code makes any sense to them?  On Windows 10, before and after handle count are identical.   On my Windows 8.1, the after count is one higher than the before count.  

[TestCode.txt](https://github.com/PowerShell/openssh-portable/files/1894624/TestCode.txt)

Reply:
@NoMoreFood , I will try to find someone to look into it. I will comment it out for now